rdr_service/alembic/ppsc/versions/509b96909dd1_attribution_table_fix.py

```diff
@@ -0,0 +1,209 @@
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('attribution_event',
+    sa.Column('id', sa.BigInteger(), autoincrement=True, nullable=False),
+    sa.Column('created', rdr_service.model.utils.UTCDateTime(), nullable=True),
+    sa.Column('modified', rdr_service.model.utils.UTCDateTime(), nullable=True),
+    sa.Column('participant_id', sa.BigInteger(), nullable=True),
+    sa.Column('event_id', sa.BigInteger(), nullable=True),
+    sa.Column('event_type_name', sa.String(length=128), nullable=True),
+    sa.Column('event_authored_time', rdr_service.model.utils.UTCDateTime(), nullable=True),
+    sa.Column('data_element_name', sa.String(length=512), nullable=True),
+    sa.Column('data_element_value', sa.String(length=512), nullable=True),
+    sa.Column('ignore_flag', mysql.TINYINT(), nullable=True),
+    sa.Column('ignore_reason', sa.String(length=512), nullable=True),
+    sa.Column('is_correction_flag', mysql.TINYINT(), nullable=True),
+    sa.Column('dev_note', sa.String(length=512), nullable=True),
+    sa.ForeignKeyConstraint(['event_id'], ['ppsc.participant_event_activity.id'], ),
+    sa.ForeignKeyConstraint(['participant_id'], ['ppsc.participant.id'], ),
+    sa.PrimaryKeyConstraint('id'),
+    schema='ppsc'
+    )
+    op.create_index(op.f('ix_ppsc_attribution_event_created'), 'attribution_event',
+                    ['created'], unique=False, schema='ppsc')
+    op.create_index(op.f('ix_ppsc_attribution_event_data_element_name'), 'attribution_event',
+                    ['data_element_name'], unique=False, schema='ppsc')
+    op.create_index(op.f('ix_ppsc_attribution_event_data_element_value'), 'attribution_event',
+                    ['data_element_value'], unique=False, schema='ppsc')
+    op.create_index(op.f('ix_ppsc_attribution_event_event_authored_time'), 'attribution_event',
+                    ['event_authored_time'], unique=False, schema='ppsc')
+    op.create_index(op.f('ix_ppsc_attribution_event_event_type_name'), 'attribution_event',
+                    ['event_type_name'], unique=False, schema='ppsc')
+    op.drop_constraint('consent_event_ibfk_1', 'consent_event', type_='foreignkey')
+    op.drop_constraint('consent_event_ibfk_2', 'consent_event', type_='foreignkey')
+    op.create_foreign_key(None, 'consent_event', 'participant_event_activity', ['event_id'],
+                          ['id'], source_schema='ppsc', referent_schema='ppsc')
+    op.create_foreign_key(None, 'consent_event', 'participant', ['participant_id'], ['id'],
+                          source_schema='ppsc', referent_schema='ppsc')
+    op.drop_constraint('deactivation_event_ibfk_1', 'deactivation_event', type_='foreignkey')
+    op.drop_constraint('deactivation_event_ibfk_2', 'deactivation_event', type_='foreignkey')
+    op.create_foreign_key(None, 'deactivation_event', 'participant_event_activity', ['event_id'],
+                          ['id'], source_schema='ppsc', referent_schema='ppsc')
```

Comment:
Is there anything we can do about this getting created on our PPSC migrations? I have to continuously clean this up also.

Reply:
Yeah, we had the same issue with NPH. I'll have to look into how that was fixed.